core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -649,6 +651,7 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
                                 parentContext: RawlsRequestContext = ctx
   ): Future[Workspace] = {
     assertBillingProfileCreationDate(profile)
+    validateWorkspaceRequest(workspaceRequest)
```

Comment:
I wish `createMultiCloudWorkspace` was called `createAzureWorkspace`, since we do plan to move to a world where a "multi-cloud"/WSM workspace could be a GCP one. It feels weird that this method assumes the workspace is Azure.

Reply:
We may not support the `authorizationDomain` field in GCP workspaces once they're moved over to WSM either as it would be best for auth domains on all WSM workspaces to be implemented with policy, so one could argue that this actually isn't Azure specific. Still, I'll look at renaming it to `createAzureWorkspace` for clarity and we can always change it back once GCP workspaces are in WSM

Reply:
I don't feel that strongly about it… `createMultiCloudWorkspace` has always assumed Azure. But every time I look at the method I have to think about it, as both the method name and the documentation lead one to believe that it is not cloud-specific.